test/X25519KeyAgreementKey2019.spec.js

```diff
@@ -89,6 +123,32 @@ describe('X25519KeyAgreementKey2019', () => {
     });
   });
 
+  describe(`export`, () => {
+    it('should export only the public key', async () => {
+      const key = await X25519KeyAgreementKey2019.generate();
+
+      const result = key.export({publicKey: true});
+      expect(result.publicKeyBase58).to.exist;
+      expect(result.privateKeyBase58).to.not.exist;
```

Comment:
These sorts of assertions can be simplified with the `keys` API. https://www.chaijs.com/api/bdd/#method_keys
```
      expect(result).to.have.keys(['publicKeyBase58']);
```

Reply:
@mattcollier ahhh yeah, I always forget about that one! Thanks.